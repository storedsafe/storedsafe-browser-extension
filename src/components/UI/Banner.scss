@import '../../variables';

.banner {
  background-color: $color-bg-banner;
  padding: $spacing * 2;
  width: 100%;
}
