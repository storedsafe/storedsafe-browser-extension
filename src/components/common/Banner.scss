@import '../../variables';

.banner {
  background-color: $color-bg-banner;
  padding: $spacing * 1.5;
  min-width: max-content;
}
