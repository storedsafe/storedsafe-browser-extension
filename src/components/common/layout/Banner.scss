@import '../../../styles/variables.scss';

.banner {
  background-color: $color-bg-banner;
  padding: $spacing * 1.5;
  min-width: max-content;
}
