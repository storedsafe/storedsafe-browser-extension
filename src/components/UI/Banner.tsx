import React from 'react';
import { Logo } from './Logo';
import './Banner.scss';

export const Banner: React.FunctionComponent = () => (
  <div className="banner">
    <Logo />
  </div>
);
