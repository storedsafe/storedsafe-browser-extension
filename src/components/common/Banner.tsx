import React from 'react'
import { Logo } from './Logo'
import './Banner.scss'

interface BannerProps {
  children?: React.ReactNode
}

export const Banner: React.FunctionComponent<BannerProps> = ({
  children
}: BannerProps) => (
  <div className="banner">
    <Logo />
    {children}
  </div>
)
