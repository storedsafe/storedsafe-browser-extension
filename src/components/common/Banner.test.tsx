import * as React from 'react';
import { mount } from 'enzyme';
import { Banner } from './Banner';

test('<Banner />', () => {
  const wrapper = mount(<Banner />);
  expect(wrapper).toMatchSnapshot();
});
