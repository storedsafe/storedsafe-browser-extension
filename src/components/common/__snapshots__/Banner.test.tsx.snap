// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Banner /> 1`] = `ReactWrapper {}`;
